src/main/java/org/opentripplanner/standalone/GrizzlyServer.java

```diff
@@ -131,7 +136,8 @@ public void run() {
         } catch (InterruptedException ie) {
             LOG.info("Interrupted, shutting down.");
         }
-        httpServer.shutdown();
```

Comment:
I'm confused about this bottom set of changes. It seems to me that we should just keep the hook, and let it happen at the bottom of run().